src/routes/landing/components/CityEntry.js

```diff
@@ -32,21 +33,17 @@ type PropsType = {|
 |}
 
 class CityEntry extends React.PureComponent<PropsType> {
+  getMunicipality = (city, filterText): Array<CityModel> => {
+    if (city._aliases && filterText) {
+      return Object.keys(city._aliases)
+        .filter(municipality => normalize(municipality).toLowerCase().includes(filterText.toLowerCase()))
```

Comment:
I guess you have to normalize the filter text to. Otherwise searching for `Königsbrunn` does not work, since the municipality key gets normalized to `Konigsbrunn`